src/test/kotlin/no/skatteetaten/aurora/databasehotel/service/DatabaseInstanceTest.kt

```diff
@@ -45,11 +51,22 @@ private fun DatabaseInstance.createDefaultSchema() = this.createSchema(defaultLa
 abstract class AbstractDatabaseInstanceTest {
 
     lateinit var instance: DatabaseInstance
+    var createdSchemas = arrayListOf<Schema>()
```

Comment:
Er det noen god grunn for å bruke ArrayList over MutableList? Jeg har oppfattet det slik at MutableList er mer idiomatisk Kotlin, men tenker du noe annet? :) I praksis så får man jo en ArrayList for begge, men det handler mer om at kotlin har abstrahert bort ArrayList for MutableList og man sier man egentlig ikke bryr seg om hvilken Collection type man får, så lenge den er Mutable.

Reply:
Jeg har ingen god grunn til at jeg brukte ArrayList, det sitter bare med gratis leie i muskelminnet :cold_sweat: 
Ellers var min oppfatning at arrayListOf og mutableListOf var likestilte i den forstand at de abstraherer bort instansiering av en collection, men her er ikke typen viktig så jeg endrer til MutableList.

Reply:
Ser den :) 
Er vel jeg som er skadet etter å ha jobbet lite med java og mye med kotlin